credential-exchange-format/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/), and this project
+adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+
+- Support for `SharedExtension` (#80)
```

Comment:
This addition is also a breaking change since it adds a variant to the `Extension` enum. Hence the `non_exhaustive` change.